src/third_party/wiredtiger/src/meta/meta_ckpt.c

```diff
@@ -240,6 +244,217 @@ __ckpt_compare_order(const void *a, const void *b)
 	return (ackpt->order > bckpt->order ? 1 : -1);
 }
 
+/*
+ * __ckpt_parse_attribute_value --
+ *
+ *  Parse the provided attribute and store the attribute value into
+ *  the appropriate ckpt variable.
+ */
+static int
+__ckpt_parse_attribute_value(
+	WT_SESSION_IMPL *session, WT_CKPT *ckpt, uint64_t attr_idx,
+	const char *val)
+{
+	uint64_t len;
+
+	/* It is possible for the addr to be empty. */
+	len = 0;
+	if (val != NULL)
+		len = strlen(val);
+
+	switch (attr_idx) {
+		case 0:
```

Comment:
It might be clearer to have some defines for each value that represent the case like:
#define CKPT_ATTR_ADDR 0

After that, you could also initialize the attribute array with this to make it even clearer:
http://gcc.gnu.org/onlinedocs/gcc-4.1.2/gcc/Designated-Inits.html

Reply:
Yes that is better. Let me make that change.

Reply:
This is done.